doc/dev/TESTING.md

```diff
@@ -1,77 +1,117 @@
 # Testing
 
-F3D has an expansive suite of tests, you may want to run them locally,
-either to validate your build or because you are contributing to F3D and want to add/modify a test.
+F3D has an expansive suite of tests that can be run locally, either to validate your build or because you are contributing to F3D and want to add/modify a test.
+To enable the tests, run `cmake -DBUILD_TESTING=ON ../` from the `build`  directory.
```

Comment:
you may want to add that to the getting started guide to help beginners if not already, but IMO this should stay in CMake options.